### PR TITLE
Agoric cli intro

### PR DESCRIPTION
**Pull request by DavidBruant**

### CODE REVIEW

File: main/getting-started/README.md [hunk 1]
```diff
@@ -68,52 +41,77 @@ yarn build
...
+
 ## Your First Agoric Dapp
 
 ![The Browser Environment](./assets/browser-environment.svg)
 
-### 1. Creating a project directory
-Let's install the default smart contracts and UI.
+In this guide, we're installing, running and interacting with the default smart contracts and UI.
+
+TODO: define/explain the various parts
```
Comment (DavidBruant) [inline main/getting-started/README.md]: The various parts of the diagram are not clear to me. I'm also in doubt with the edges with "HTML"

File: main/getting-started/README.md [hunk 2]
```diff
@@ -68,52 +41,77 @@ yarn build
...
-```sh
 # Install Javascript dependencies.
 agoric install
 ```
 
-### 3. Launching the Agoric server
-Next, let's start up the Agoric VM. This creates the "vats" in which
-our smart contract code will be run.
+
+### Start the local Agoric Node, wallet server and Dapp Server
```
Comment (DavidBruant) [inline main/getting-started/README.md]: No clue if i've listed them all properly

File: main/getting-started/README.md [hunk 3]
```diff
@@ -68,52 +41,77 @@ yarn build
...
 # Run the local vat machine.
 agoric start
 ```
 
-### 4. Installing a contract
-Let's deploy our dapp on the Agoric VM.
+Keep this terminal open somewhere
+
+You can now open
+- [http://localhost:3000](http://localhost:3000) to see our demo DApp
```
Comment (DavidBruant) [inline main/getting-started/README.md]: I haven't been able to see anything at this address

File: main/getting-started/README.md [hunk 4]
```diff
@@ -68,52 +41,77 @@ yarn build
...
 ```sh
+# make sure you're in the 'demo' directory
+
 # Install your smart contract and web api (can be done separately)
 agoric deploy ./contract/deploy.js ./api/deploy.js
 ```
+
+TOCLARIFY: why those 2 files?
+What does each refer to?
+What is the benefit of deploying together or separately?
```
Comment (DavidBruant) [inline main/getting-started/README.md]: questions

File: main/getting-started/README.md [hunk 5]
```diff
@@ -68,52 +41,77 @@ yarn build
...
+
+
 And navigate to our wallet
 [http://localhost:8000/wallet/](http://localhost:8000/wallet/)
 
-### 5. Launching a demo DApp server
+TOCLARIFY: what is the user supposed to see/noticed/understand with this step?
+They deployed a contract. Who are the parties of this contract? Does the user play one or both roles (or none)?
```
Comment (DavidBruant) [inline main/getting-started/README.md]: questions

File: main/getting-started/README.md (hunk 3 above)
Comment (Chris-Hibbert) [reply main/getting-started/README.md]: I think :3000 is a typo, and the :8000 below is correct.

File: main/getting-started/README.md (hunk 5 above)
Comment (Chris-Hibbert) [reply main/getting-started/README.md]: I think this run-through happens in the context of running on a single machine, so the user is playing all the roles. They can interact with the contract as its owner while using the wallet and exchange UIs as an end-user.  We should be clearer about which role is played by the user in each window/REPL/CLI. 

We have other variants of the diagram above that are clearer about which process/vat/chain is acting on behalf of which party.

**Comment by katelynsills** [comment]
A lot of these changes look good to me. I get the feeling that this instructions will be changing soon. Let's hold off on merging until we have some more clarity on what will stay the same.

**Comment by katelynsills** [comment]
@michaelfig I'm not sure how this PR would affect your current plans for the agoric-cli. Can we talk it through during the Zoe/Alpha meeting?

**Comment by katelynsills** [comment]
@DavidBruant is this PR still applicable?

**Comment by katelynsills** [comment]
Closing as outdated

**Pull request closed without merging**